### DISCUSSION TITLE
PP-OCRv4识别模型

### DISCUSSION THREAD

**Discussion by hezhenname** [PaddlePaddle/PaddleOCR#12487] [Q&A]
为什么同样一张图片，在调用高精度识别模型之后，需要5分钟左右得出最后的结果，而超轻量模型只需不到20秒，速度差别这么大吗

**Comment by Plz-Thx** [comment]
我也碰到了类似的问题：
在仅运行rec模型的情况下，使用FastDeploy框架在win+AMD（R7）的CPU推理v4的server版onnx模型大概100ms，但换成win+ Intel（i7）的cpu后就增加到近4000ms。
反而PaddleOCRv4版在两个环境下的推理速度几乎没区别，已经确定不是依赖库（onnxruntime，opencv等）导致的，所以大概率是server模型的问题？还请百度的同学看一下这个奇怪的问题

**Comment by nissansz** [comment]
> 我也碰到了类似的问题： 在仅运行rec模型的情况下，使用FastDeploy框架在win+AMD（R7）的CPU推理v4的server版onnx模型大概100ms，但换成win+ Intel（i7）的cpu后就增加到近4000ms。 反而PaddleOCRv4版在两个环境下的推理速度几乎没区别，已经确定不是依赖库（onnxruntime，opencv等）导致的，所以大概率是server模型的问题？还请百度的同学看一下这个奇怪的问题

解决了吗？
